frontend/app/views/spree/products/_cart_form.html.erb

```diff
@@ -1,84 +1,79 @@
-<%= form_for :order, html: { id: 'add-to-cart-form' } do |f| %>
-  <div class="row" id="inside-product-cart-form" data-hook="inside_product_cart_form" itemprop="offers" itemscope itemtype="https://schema.org/Offer">
-    <% if @product.variants_and_option_values(current_currency).any? %>
-      <div id="product-variants" class="col-lg-6 mt-4">
-        <h3 class="product-section-title"><%= Spree.t(:variants) %></h3>
-        <ul class="list-group">
-          <% @product.variants_and_option_values(current_currency).each_with_index do |variant, index| %>
-            <li class="list-group-item <%= "active" if index == 0 %>">
-              <div class="form-check">
-                <%= radio_button_tag "variant_id", variant.id, index == 0,
-                    'data-price' => variant.price_in(current_currency).money,
-                    'data-in-stock' => variant.can_supply?,
-                    'data-backordered' => variant.backordered?,
-                    class: 'form-check-input'
-                %>
-                <%= label_tag "variant_id_#{ variant.id }", class: "form-check-label" do %>
-                  <span class="variant-description">
-                    <%= variant.options_text %>
-                  </span>
-                  <% if variant_price variant %>
-                    <span class="price diff"><%= variant_price variant %></span>
-                  <% end %>
-                  <% unless variant.can_supply? %>
-                    <span class="out-of-stock"><%= Spree.t(:out_of_stock) %></span>
-                  <% end %>
-                <% end %>
-              </div>
-            </li>
-          <% end%>
-        </ul>
+<template class="availability-template availability-template-not-available-in-currency">
+  <%= render 'cart_form_availability_templates', type: 'not-available-in-currency' %>
+</template>
+
+<template class="availability-template availability-template-in-stock">
+  <%= render 'cart_form_availability_templates', type: 'in-stock' %>
```

Comment:
do we have this partial in the codebase?

Reply:
yep https://github.com/spark-solutions/spree/blob/0d834689561bd19459c668c2d23fca61489bfff8/frontend/app/views/spree/products/_cart_form_availability_templates.html.erb